pep-0570.rst

```diff
@@ -221,7 +327,7 @@ remains the same as the one for positional-or-keyword. Once
 a positional-only argument is provided with a default,
 the following positional-only and positional-or-keyword argument
 needs to have a default as well. Positional-only parameters that
-don’t have a default value are "required" positional-only parameters.
+do not have a default value are "required" positional-only parameters.
```

Comment:
```suggestion
do not have a default values are *required* positional-only parameters.
```

Quotes here makes sense but italics has the same meaning and disambiguates between quoting it as a specific term and quoting it as scare quotes (which would indicate that it's a soft or often ignored requirement).